content/en/docs/concepts/scheduling-eviction/scheduling-framework.md

```diff
@@ -130,6 +130,15 @@ defined range of integers representing the minimum and maximum scores. After the
 [NormalizeScore](#normalize-scoring) phase, the scheduler will combine node
 scores from all plugins according to the configured plugin weights.
 
+#### Capacity scoring {#scoring-capacity}
+
+{{< feature-state feature_gate_name="StorageCapacityScoring" >}}
+
+Nodes can be scored based on their available storage capacity. This only applies
+when not using static provisioning
+([dynamic provisioning](/docs/concepts/storage/dynamic-provisioning/) only), and
+this feature is currently in alpha.
```

Comment:
Early feedback: I think this detail would belong in https://kubernetes.io/docs/reference/scheduling/config/#scheduling-plugins

Reply:
Thank you. I will address it once the work on [the k/k PR](https://github.com/kubernetes/kubernetes/pull/128184) is complete.

Reply:
I will add to that document once the feature gate is enabled by default.